ShvetsovAM/lab1/Calculator/Calculator/Calculator.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+#include <conio.h>
+using namespace std;
+int main()
+{
+	setlocale(0, "");
+	float first, second, result;
+	char sign;
+	cout << "������� ������ �����";
+	cin >> first;
+	cout << "������� ������ �����";
+	cin >> second;
+	cout << "������� ����";
+	cin >> sign;
+
+	switch (sign)
+	{
+	case '+': result = first + second;
+		break;
+	case '-': result = first - second;
+		break;
+	case '*': result = first*second;
+		break;
+	case '/': result = first / second;
+
+		if (second == 0)
+		{
+			cout << "������: �� ���� ������ ������";
+			_getch();
+			return 0;

```

Comment:
Do not use return statement here
Normally you don't wont calculator you use to shut down every time you make a mistake